### PR TITLE
Updated sentencepiece to generate TF plugin library and refreshed to …

### PR DESCRIPTION
**Pull request by npanpaliya**
…the latest release

### CODE REVIEW

File: conda-recipes/sentencepiece-feedstock/recipe/build.sh [hunk 1]
```diff
@@ -17,6 +17,47 @@
...
+  if [ "$1" != "" ]; then
+    pkg_name="==$1"
+  fi
+
+  export IBM_POWERAI_LICENSE_ACCEPT=yes
+  conda create -n tf$1 -y python=3.6
+
+  conda activate tf$1
+  conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+  conda config --prepend channels http://ausgsa.ibm.com/projects/p/powerai-conda/conda-early-access/
```
Comment (wdirons) [inline conda-recipes/sentencepiece-feedstock/recipe/build.sh]: We can't reference GSA in external github. Instead use: https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda-early-access/

Comment (npanpaliya) [reply conda-recipes/sentencepiece-feedstock/recipe/build.sh]: Yeah. I've removed that. Thanks Billy!

**Comment by wdirons** [comment]
Does this need to be built using tensorflow-gpu? Or could it be built using the tensorflow for cpu container and conda package? That would work around the libcuda.so.1 error and probably lead to faster build times if your reinstall tensorflow for several versions of it.

**Comment by npanpaliya** [comment]
Actually tensorflow-gpu is installed only for 1.13.1 as we didn't have tensorflow CPU only version for 1.13.1. For later versions, it is installed TF CPU only as we are not explicitly installing `tensorflow-gpu` in the script. 
I'm actually confused how does it work locally when I try to build sentencepiece with this change on the same docker container. Even apart from builds, if I try to install TF 1.13.1 on the same container, it works without any complaint for libcuda.so.1.
For now, I've tried exporting LD_LIBRARY_PATH as libcuda.so.1 is at /usr/lib/powerpc64le-gnu-linux on this container.

**Comment by wdirons** [comment]
i think the difference is your local system has GPUs while the travis ci environment does not. @jayfurmanek , do we even need to support sentencepiece for TF 1.13.1? Or is 1.14 and later good enough?

**Comment by npanpaliya** [comment]
@jayfurmanek - For now, I've disabled TF shared lib to be created for 1.13.1 and with that the build has passed. Please let me know if it's fine to drop TF .1.13.1 since  it's TF-gpu.

**Comment by jayfurmanek** [comment]
I'm with leaving TF 1.13 off. This looks good to me.

**Comment by npanpaliya** [comment]
@wdirons - Please give your approval as I can't merge without it.

**Pull request merged**